libcontainer/specconv/spec_linux.go

```diff
@@ -981,8 +979,10 @@ func parseMountOptions(options []string) *configs.Mount {
 		if f, exists := mountFlags[o]; exists && f.flag != 0 {
 			if f.clear {
 				m.Flags &= ^f.flag
+				m.ClearedFlags |= f.flag
```

Comment:
Maybe add a comment here?

Reply:
I think the docstring in configs is sufficient. The meaning of the field is pretty self-explanatory.